### PR TITLE
Translatable Markdown Editor (and Blog Markdown Editor)

### PR DESCRIPTION
**Pull request by rsobon**
I've created multilingual form widgets for markdown (it is working OK with the rainlab/blog plugin and normal markdown fields)

There were some conflicts in the CSS between markdown editor's `.btn`, `.dropdown-menu`  and the language selector's button so I renamed some styling and did refactoring.

Also there is small problem with the `dropdown.js` from the october storm library. Because it was conflicting with the "formatting" dropdown (`data-dropdown-title="Formatting"`) in the markdown editor. I made the pull request in the `octobercms/october` repo with the proper fix (pull request here https://github.com/octobercms/october/pull/1933)

### CODE REVIEW

**Comment by rsobon** [comment]
Hello, any progress on merging this?

File: formwidgets/mlricheditor/assets/js/mlricheditor.js [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
         $el.on('setLocale.oc.multilingual', function(e, locale, localeValue) {
             if (typeof localeValue === 'string' && $richeditor.data('oc.richEditor')) {
-                $richeditor.richEditor('setContent', localeValue);
```
Comment (daftspunk) [inline formwidgets/mlricheditor/assets/js/mlricheditor.js]: Redactor is no longer used, this line should be reverted

File: Plugin.php [hunk 2]
```diff
@@ -162,6 +162,14 @@ public function registerFormWidgets()
...
                 'label' => 'Rich Editor (ML)',
                 'code'  => 'mlricheditor'
+            ],
+            'RainLab\Translate\FormWidgets\MLBlogMarkdown' => [
+                'label' => 'Blog Markdown Editor (ML)',
+                'code'  => 'mlblogmarkdown'
+            ],
+            'RainLab\Translate\FormWidgets\MLMarkdownEditor' => [
+                'label' => 'Markdown Editor (ML)',
+                'code'  => 'mlmarkdowneditor'
```
Comment (daftspunk) [inline Plugin.php]: Why the need for two widgets?

**Comment by daftspunk** [comment]
Merged in 30d90801162ea69ba26063215a995bf987847aea
Thanks!

**Comment by marcoelho97** [comment]
How do I use this??

**Pull request closed without merging**